WcaOnRails/db/migrate/20160514141051_fill_accepted_at_registrations_column.rb

```diff
@@ -2,7 +2,7 @@ class FillAcceptedAtRegistrationsColumn < ActiveRecord::Migration
   def change
     # Move the data
     Registration.where(status: "a").includes(:competition).each do |registration|
-      registration.update_column(:accepted_at, registration.competition.registration_open)
+      registration.update_column(:accepted_at, registration.competition.registration_open) if competition
```

Comment:
I almost asked about this, but then I saw https://github.com/cubing/worldcubeassociation.org/pull/628 =)